tests/unit/libbpf_test.cpp

```diff
@@ -2967,8 +2969,10 @@ TEST_CASE("sequential_tail_call", "[libbpf]")
     }
 
     // Verify that the printk output is correct.
-    REQUIRE(output.size() == MAX_TAIL_CALL_CNT);
-    for (size_t i = 0; i < MAX_TAIL_CALL_CNT; i++) {
+    // In sequential ebpf program, the printk is added even to the caller.
```

Comment:
I can't parse "In sequential ebpf program"

Reply:
In tail_call_sequence.c
SEC("xdp_prog" #X) int sequential##X(struct xdp_md* ctx)

Reply:
Update comment to clarify

Reply:
Fixed